temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -437,7 +437,9 @@ def _public_callables(parent: Any, *, exclude: Set[str] = set()) -> Set[str]:
         # TODO(cretz): Fix issues with class extensions on restricted proxy
         # "argparse": SandboxMatcher.all_uses_runtime,
         "bz2": SandboxMatcher(use={"open"}),
-        "concurrent": SandboxMatcher.all_uses_runtime,
+        "concurrent": SandboxMatcher(
+            children={"futures": SandboxMatcher.all_uses_runtime}
+        ),
```

Comment:
We should allow using the concurrent.futures API only at import time.

Reply:
That's what this does. But before I didn't even allow it to be imported.